openmc/mesh.py

```diff
@@ -6,6 +6,8 @@
 import numpy as np
 
 import openmc.checkvalue as cv
+import openmc.surface
```

Comment:
Do these need to be imported individually? You can access `Cell` and `Surface` directly from the `openmc` namespace.